### PR TITLE
Fix Litepicker rendering issue

### PR DESCRIPTION
**Pull request by Peartes**
This commit (thanks Doug) fixes the binding issue lite picker has and also validates the end and start date 
to make sure that the end date is after the start date

These are the issues: [issue](https://www.notion.so/curvelabs/339a249185e14ddcbd731e5ba07c46c7?v=f7cba822bb9043a5a87ab80b5e2b40ef&p=72b3b1a660ce4fd7925316964f9391aa) resolved and [issue](https://www.notion.so/curvelabs/339a249185e14ddcbd731e5ba07c46c7?v=f7cba822bb9043a5a87ab80b5e2b40ef&p=2253eda597c5492094ff85365ef39429)

### CODE REVIEW

File: src/newSeed/stage4.html [hunk 1]
```diff
@@ -86,7 +87,8 @@
                 <input
                   id="startTime" type="time"
                   value.bind="startTime"
-                  placeholder="HH:MM (GMT)" step="60"></input>
+                  placeholder="HH:MM (GMT)" step="60"
+                  autocomplete="off"></input>
```
Comment (dkent600) [inline src/newSeed/stage4.html:91]: should not be in this PR

File: src/newSeed/stage4.html [hunk 2]
```diff
@@ -114,7 +117,8 @@
               <div class="vestingDates">
                 <input
                   id="endTime" value.bind="endTime"
-                  placeholder="HH:MM (GMT)" type="time" step="60"></input>
+                  placeholder="HH:MM (GMT)" type="time"
+                  autocomplete="off"></input>
```
Comment (dkent600) [inline src/newSeed/stage4.html:121]: should not be in this PR

File: src/newSeed/stage4.html [hunk 3]
```diff
@@ -75,7 +75,8 @@
                 placeholder="YYYY-MM-DD"
                 class="vestingDateInput"
                 value.bind="startDate | date"
-                ref="startDateRef" />
+                ref="startDateRef" 
+                autocomplete="off" disabled/>
```
Comment (dkent600) [inline src/newSeed/stage4.html:79]: why disabled?

File: src/newSeed/stage4.html [hunk 4]
```diff
@@ -104,7 +106,8 @@
                 class="vestingDateInput"
                 value.bind="endDate | date"
                 placeholder="YYYY-MM-DD"
-                ref="endDateRef" />
+                ref="endDateRef" 
+                autocomplete="off" disabled/>
```
Comment (dkent600) [inline src/newSeed/stage4.html:110]: why disabled?

**Review by dkent600 — CHANGES_REQUESTED** [review]

File: src/newSeed/stage4.html (hunk 4 above)
Comment (Peartes) [reply src/newSeed/stage4.html:110]: To disallow users from inputting into the input field

File: src/newSeed/stage4.html [hunk 5]
```diff
@@ -117,8 +116,7 @@
               <div class="vestingDates">
                 <input
                   id="endTime" value.bind="endTime"
-                  placeholder="HH:MM (GMT)" type="time"
-                  autocomplete="off"></input>
+                  placeholder="HH:MM (GMT)" type="time" step="60"></input>
```
Comment (dkent600) [inline src/newSeed/stage4.html]: type=time is still in there.  why is `step` in there?

**Review by dkent600 — CHANGES_REQUESTED** [review]

**Review by dkent600 — APPROVED** [review]

**Pull request merged**